certified-connectors/Luware Nimbus/apiProperties.json

```diff
@@ -0,0 +1,94 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<< Please add your client id >>",
+          "scopes": [ "User.Read" ],
+          "redirectMode": "Global",
```

Comment:
Can you please change is to "redirectMode": "GlobalPerConnector", please refer this link https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#verify-connection-parameters

Reply:
Thanks for the corrections. I have updated the PR with all suggested changes.